sys/kern/subr_intr.c

```diff
@@ -352,6 +372,12 @@ intr_irq_handler(struct trapframe *tf)
 #endif
 }
 
+void
+intr_irq_handler(struct trapframe *tf)
+{
+	intr_irq_handler_root(tf, INTR_ROOT_IRQ);
```

Comment:
I believe FreeBSD style is there should be a blank line between these two.

Reply:
Not since 2019

Reply:
ok left it as-is